config/example/config.txt

```diff
@@ -151,12 +151,6 @@ TRAITOR_SCALING
 ## If security is prohibited from being most antagonists
 #PROTECT_ROLES_FROM_ANTAGONIST
 
-## Uncomment this to DISABLE persistence

```

Comment:
we may as well throw these extra settings in if we need them later

Reply:
I restored them in the example.